.gitignore

```diff
@@ -4,6 +4,6 @@ _site/
 .jekyll-cache/
 .bundle/
 node_modules/
-build/bundle.js*
+build/bundle*
```

Comment:
do we want to disregard bundle.min.js?

Reply:
I think so, since it causes merge conflicts and we have a workflow to recompile it when we merge any PR

Reply:
makes sense 👍 if that's the case, can we delete the checked-in `bundle.min.js` file (so that we don't track that version either when we pull -- i'm not sure if that's how it behaves tho, will have to see haha) in the same PR?